remix/app/config/db.server.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from "@prisma/client";
+
+let db: PrismaClient;
```

Comment:
That code below looks complex. Do we need it all? Can we shorten it to:

```javascript
import { PrismaClient } from '@prisma/client';

const dbClient = new PrismaClient();

export default dbClient;
```

Reply:
My code below is come from Remix suggestion that I follow them at the beginning, but I think follow your way (that also suggestion from prisma) I think is better.

Fixed